tutorial/cloudshell_tutorial.md

```diff
@@ -25,7 +25,7 @@ If you'd prefer to follow this tutorial on your local machine, you can follow [t
 
 Terraform is already install in your Cloud Shell environment. You can verify this by running:
 
-```shell
+```bash
```

Comment:
`bash` will be recognised as a command and appear like this:

![image](https://user-images.githubusercontent.com/2080520/63113523-90892100-bf47-11e9-9549-08c0c15aa792.png)

The shell icon will copy the command to the shell directly, making it super easy to run commands.